ecosystem/sep-0024.md

```diff
@@ -190,6 +192,127 @@ The deposit flow can only be fulfilled if the Stellar `Account` has established
 5. `Anchor` detects the trust line creation in the `Account`. If the asset is `AUTH_REQUIRED`, `Anchor` approves the new trust line.
 6. `Anchor` proceeds with the deposit flow.
 
+##### Claimable Balances
+
+[Claimable Balances](https://www.stellar.org/blog/protocol-14-improvements) enables payments to accounts that do not have a trust line for the asset being sent.
+This feature provides less friction for Stellar accounts who wish to make an offchain deposit to anchors without making a trust line before the deposit is complete.
+Anchors will no longer have to wait during the time when the account doesn't have a trust line. Instead, anchors can make the payment using ClaimableBalance operation and the stellar account (wallet user) can collect the funds at their own convenience.
+
+**NOTE**: The Stellar Account will still need to establish trust line to the asset before claiming the outstanding balance.
```

Comment:
maybe this belongs in the `Wallet Feature: Claim Account Claimable Balance` section? It seems a bit early to say this here.